src/app/views/App.tsx

```diff
@@ -86,7 +86,7 @@ class App extends Component<IAppProps, IAppState> {
 
   public componentDidMount = async () => {
     this.displayToggleButton(this.mediaQueryList);
-    this.mediaQueryList.addListener(this.displayToggleButton);
+    this.mediaQueryList.addEventListener('click',this.displayToggleButton);
```

Comment:
Using devtools I can set browser to different phones. 
What I have noticed is that the app doesn't know that it's in mobile view so it does not respond appropriately.

I think the culprit could be setting the listener to listen to a 'click' event. I am not sure which one it should be.

Reply:
Ah, this is a great observation. 
Okay, let me look this up... 
There's probably a more appropriate listener